### DISCUSSION TITLE
equalant of AWS4Auth from "requests_aws4auth" package in httpx

### DISCUSSION THREAD

**Discussion by blnprasad** [encode/httpx#2141] [Q&A]
Hi,
     I'm looking for solution for async version of AWS4Auth  from "requests_aws4auth" package. any suggestions on same if I want to replace "from requests.auth import AuthBase" with async version from httpx?

**Comment by johtso** [comment]
This looks like the equivalent for httpx: https://github.com/Colin-b/httpx_auth#aws-signature-v4

Please use discussions for these kinds of questions.

**Comment by blnprasad** [comment]
> This looks like the equivalent for httpx: https://github.com/Colin-b/httpx_auth#aws-signature-v4
> 
> Please use discussions for these kinds of questions.

Thank you. sorry for raising here. will use discussions.
